pkg/settings/settings.go

```diff
@@ -154,6 +157,10 @@ func (s Setting) GetInt() int {
 	return i
 }
 
+func (s Setting) GetDuration() time.Duration {
+	return time.Duration(s.GetInt())
```

Comment:
If the setting is cleared, s.GetInt() returns 0, will this function return the default value? And, this value should have a minimal value, user can't set it as 1, 2 ...; for the max value, a limitation seems also required.

BTW, with a new `setting`, the Harvester document needs to updated, do we have another PR for it ? thanks.

Reply:
Currently, `GetInt` is implemented like below.

```go
func (s Setting) GetInt() int {
	v := s.Get()
	i, err := strconv.Atoi(v)
	if err == nil {
		return i
	}
	logrus.Errorf("failed to parse setting %s=%s as int: %v", s.Name, v, err)
	i, err = strconv.Atoi(s.Default)
	if err != nil {
		return 0
	}
	return i
}
```

So, if the setting is cleared, `GetInt` returns default value and `GetDuration` also return default value. 

It seems there is no any existing mechanism to add this restriction of minimum and maximum, but I think it's good to add it. Maybe we could open another enhancement issue to track this? Because I think it would be better to check other settings's minimum and maximum if it's necessary.

Regarding document, thanks for reminding. I'll check this with @bk201.

Reply:
The value limitation is more checked in webhook.

Reply:
```
	v := s.Get()
	i, err := strconv.Atoi(v)
```

Check `v == ""` first; otherwise, it will print the error